bin/cqlsh.py

```diff
@@ -1108,14 +1115,17 @@ def parse_for_update_meta(self, query_string):
         return self.get_table_meta(ks, cf)
 
     def perform_simple_statement(self, statement):
+        """returns tuple of success (boolean) and a future of the result"""
         if not statement:
             return False, None
 
         future = self.session.execute_async(statement, trace=self.tracing_enabled)
         result = None
+        err = None
```

Comment:
Probably should be "traced_err" (or something similar) to make clear that it's not some kind of general error value